config/kubernetes/demo/deployment.yaml

```diff
@@ -174,3 +174,50 @@ spec:
                 name: environment-variables
             - secretRef:
                 name: app-secrets
+      initContainers:
+        - name: migrations
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/correspondence/track-a-query-ecr:latest
+          command: 
+            - rake
+            - db:abort_if_pending_migrations
+          env:
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-rds-output
+                  key: url
+            - name: CORRESPONDENCE_PLATFORM_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-rds-output
+                  key: database_password
+            - name: REDIS_URL
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-elasticache-redis-output
+                  key: url
+            - name: REDIS_AUTH_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-elasticache-redis-output
+                  key: auth_token
+            - name: SETTINGS__CASE_UPLOADS_S3_BUCKET
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-s3-output
+                  key: bucket_name
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-s3-output
+                  key: access_key_id
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: track-a-query-s3-output
+                  key: secret_access_key
```

Comment:
I don't think the migrations container requires these env vars. In the interest of reducing boilerplate, could we try without these and see if it's happy?

Reply:
OK, I'll test without. Thanks.